src/main/java/lotterymachine/utils/LotteryNumbersGenerator.java

```diff
@@ -0,0 +1,14 @@
+package lotterymachine.utils;
+
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Collectors;
+
+import static lotterymachine.utils.LotteryRule.*;
+
+public class LotteryNumbersGenerator {
+    public static List<Integer> generate() {
+         return ThreadLocalRandom.current().ints(MINIMUM.getNumber(), MAXIMUM.getNumber())
+                 .distinct().limit(TICKET_SIZE.getNumber()).boxed().collect(Collectors.toList());
+    }
```

Comment:
체이닝 메서드의 경우는 적절히 개행을 한다면 코드의 가독성을 높일 수 있습니다!!

```suggestion
    public static List<Integer> generate() {
         return ThreadLocalRandom.current()
                 .ints(MINIMUM.getNumber(), MAXIMUM.getNumber())
                 .distinct()
                 .limit(TICKET_SIZE.getNumber())
                 .boxed()
                 .collect(Collectors.toList());
    }
```

그리고 추가적으로 이러한 방법이 아닌, 이번 미션에서 제안한 `Collections.shuffle()`을 사용해서 구현해보는 것도 좋을 것 같아요 :)

<img width="517" alt="Screen Shot 2022-02-25 at 1 39 03 AM" src="https://user-images.githubusercontent.com/48052622/155567780-4e56de6f-abc3-4707-8818-fb79ef13bf21.png">

Reply:
현재는 ThreadLocalRandom을 사용하지 않고, 1~45개의 숫자를 가지는 LotteryNumber 리스트를 미리 생성하여 shuffle()후에 반환하는 형태로 변경했습니다😊